<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>371</width>
     <height>542</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>brightness</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>contrast</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSpinBox" name="spinBox_3">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_3">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>saturation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="spinBox_4">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_4">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>gain</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QSpinBox" name="spinBox_5">
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_5">
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>power_line_frequency</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QSpinBox" name="spinBox_6">
        <property name="minimum">
         <number>2000</number>
        </property>
        <property name="maximum">
         <number>6500</number>
        </property>
        <property name="value">
         <number>4000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_6">
        <property name="minimum">
         <number>2000</number>
        </property>
        <property name="maximum">
         <number>6500</number>
        </property>
        <property name="value">
         <number>4000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>white_balance_temperature</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QSpinBox" name="spinBox_7">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_7">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>sharpness</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QSpinBox" name="spinBox_8">
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_8">
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>backlight_compensation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QSpinBox" name="spinBox_9">
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_9">
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>exposure_auto</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QSpinBox" name="spinBox_10">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>2047</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_10">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>2047</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>exposure_absolute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QSpinBox" name="spinBox_11">
        <property name="minimum">
         <number>-36000</number>
        </property>
        <property name="maximum">
         <number>36000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_11">
        <property name="minimum">
         <number>-36000</number>
        </property>
        <property name="maximum">
         <number>36000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>pan_absolute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QSpinBox" name="spinBox_12">
        <property name="minimum">
         <number>-36000</number>
        </property>
        <property name="maximum">
         <number>36000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_12">
        <property name="minimum">
         <number>-36000</number>
        </property>
        <property name="maximum">
         <number>36000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>tilt_absolute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QSpinBox" name="spinBox_13">
        <property name="maximum">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_13">
        <property name="maximum">
         <number>250</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>focus_absolute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QSpinBox" name="spinBox_14">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_14">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>zoom_absolute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>16</y>
     <width>293</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>238</width>
          <height>33</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Please input the command here.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>cmd run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>318</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>The result will be shown here.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>441</y>
     <width>294</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>33</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto'; font-size:12pt; font-weight:200; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;v4l2-ctl --all&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Normal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Location_Push">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
